app/components/order-card.js

```diff
@@ -11,6 +11,15 @@ export default class OrderCard extends Component {
     return amount === null || isEqual(amount, 0);
   }
 
+  @computed('order.attendees')
+  get allTickets() {
+    const Tickets = [];
+    this.order?.attendees?.map(attendee => {
+      if (attendee?.ticket?.get('name')) {Tickets.push(attendee?.ticket)}
+    });
+    return Tickets;
+  }
```

Comment:
Not needed

Reply:
the test was failing. it was showing can't read name of undefined.

Reply:
Still this is not needed. Test could have been fixed. No need of JS if it can be done purely in template